docs/container.md

```diff
@@ -26,10 +26,10 @@ USAGE
 
 ## `heroku container:login`
 
-log in to Heroku Container Registry
+log into Heroku Container Registry
```

Comment:
I'm currently experimenting with `in to`/`into`. People have strong opinions about it. I'm happy to drop this or any other change. (Note that I didn't change all `in to`s to `into`.)

My claim is that the verb is `log` and the preposition is `into`. But others appear to disagree.